sockeye/arguments.py

```diff
@@ -741,6 +731,14 @@ def add_inference_args(params):
                                type=int_greater_or_equal(1),
                                default=5,
                                help='Size of the beam. Default: %(default)s.')
+    decode_params.add_argument('--batch-size',
+                               type=int_greater_or_equal(1),
+                               default=1,
+                               help='Size of the batch. Default: %(default)s.')
+    decode_params.add_argument('--portion-size',
```

Comment:
can we rename this to --chunk-size (and similarly in the code below: portion -> chunk)? I think this is a more common term.